web/business-registry-dashboard/app/locales/en-CA.ts

```diff
@@ -248,7 +248,7 @@ export default {
     DISSOLUTION: 'This business is in the process of being dissolved',
     PROCESSING: 'This name request is still processing. It may take up to 10 minutes.',
     EXPIRED: 'This {type} is no longer valid; the name request is expired.',
-    EXPIRING_SOON: 'This name request is expiring soon. Please Register Business.',
+    EXPIRING_SOON: 'This name request is expiring soon. Complete your application before the expiry date',
```

Comment:
Like the other messages, this should probably have a period at the end.

Ditto the FR version.

Reply:
Well, I see some other messages are inconsistent. I this the periods are correct but up to you if you want to change all entity alert types or not.

Reply:
Status quo.